src/lib/sdk.ts

```diff
@@ -110,10 +116,14 @@ class FlutterSDK {
 
       if (flutterPath) {
         flutterPath = await getRealPath(flutterPath);
+        flutterPath = flutterPath.trim();
         if (flutterPath.endsWith(join('bin', 'flutter'))) {
-          this._sdkHOme = flutterPath.replace(join('bin', 'flutter'), '');
+          this._sdkHome = flutterPath.replace(join('bin', 'flutter'), '');
+        } else {
+          this._sdkHome = flutterPath;
```

Comment:
`this._sdkHome = flutterPath` flutterPath is flutter command path, so it seems can't be sdkHome.

Reply:
Well I assumed here that if it did not end with `/bin/flutter` that the user gave use the sdk directory.
If you think we should not do that we should probably throw an error to the user that the given flutter lookup is incorrect

Reply:
OK, got it.